src/DesignKit.tsx

```diff
@@ -408,6 +408,34 @@ const ButtonsSection: React.FC = () => {
             Dashed
           </Button>
         </Space>
+        <strong>Shape variation</strong>
+        <Row>
+          Circle
+          <Button icon={<Icon name="power" title="power" />} shape="circle" />
+          Round
+          <Button icon={<Icon name="power" title="power" />} shape="round" />
+        </Row>
```

Comment:
q: what happens if you try to put text inside a `round` or `circle` button, instead of just an icon?

Reply:
I think circle looks odd if its content is a rectangle instead of a square so it probably wouldn't be used for that, but I can add examples with text to illustrate this more clearly.

Reply:
You limit the prop types so that those shapes can't contain children, but up to you.